backend/internal/dto/oidc_dto.go

```diff
@@ -31,6 +33,7 @@ type OidcClientCreateDto struct {
 	LogoutCallbackURLs []string                 `json:"logoutCallbackURLs"`
 	IsPublic           bool                     `json:"isPublic"`
 	PkceEnabled        bool                     `json:"pkceEnabled"`
+  RequiresReauthentication bool               `json:"requiresReauthentication"`
```

Comment:
Please run gofmt (you can also configure your IDE to format Go files automatically on save)